src/resources/elements/dropdown/dropdown.ts

```diff
@@ -44,10 +44,12 @@ export class Dropdown {
     /**
      * add class to each
      */
-    this.dropdownOptions = Array.from(this.menu.children);
+    this.dropdownOptions = Array.from(this.options.children);
     this.dropdownOptions.forEach((item, index) => {
-      item.classList.add("option");
-      item.addEventListener("click", (e) => this.handleOptionSelected(e, index));
+      if (item.tagName !== "HR") {
```

Comment:
I think would be better if the HR were identified as a ref and you just check here for inequality with that instead of the tag name.